.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - 2.7
+  - 3.6
+before_install:
+  - sudo add-apt-repository ppa:alex-p/tesseract-ocr
```

Comment:
Tesseract 4.0 for Ubuntu 18.04 should be coming soon, then we can switch to using apt with no ppa.

See tesseract issue [#1423](https://github.com/tesseract-ocr/tesseract/issues/1423) and recent release of [4.0.0-beta.1](https://github.com/tesseract-ocr/tesseract/releases/tag/4.0.0-beta.1)

Reply:
Problem with travis is that you can only choose between 12.04 and 14.04. At least since 16.04 / Debian 7, 3.04 is shipped and tesserocr should build.

Even with the PPA it doesn't build for me since tesseract 4 seems to make use of some c++11 features that the 14.04 gcc version doesn't support.

I'm now using the same pre-install setup that tesserocr uses (download and build leptonica and tesseract). Very inefficient but it worked: https://travis-ci.org/OCR-D/pyocrd/jobs/358810995